src/Microsoft.AspNet.Mvc.ViewFeatures/ViewResult.cs

```diff
@@ -79,6 +79,7 @@ public class ViewResult : ActionResult
                             actionContext = context,
                             result = this,
                             viewName = viewName,
+                            viewEgineType = viewEngine.GetType(),
```

Comment:
Realized I didn't answer you original question. We are wanting to let the user know which view engine made the decision... i.e. in v1 we would tell people that it was the WebForm ViewEngine that did a given action and then it went to the Razor ViewEngine. 

![viewstab](https://cloud.githubusercontent.com/assets/585619/10062721/ad67ac44-621a-11e5-833b-1db5f9c407c1.png)